### PR TITLE
exodos titles and boxarts

### PR DESCRIPTION
**Pull request by WizzardSK**

### CODE REVIEW

**Comment by RobLoach** [comment]
libretro-daatabase indexes against totaldoscollection, and it looks like the titles don't have the years on them:
https://raw.githubusercontent.com/libretro/libretro-database/master/dat/DOS.dat

If we remove the years in the filenames, I believe this could work. Thanks a lot for pushing this forwards!

Alternatively, we could append the year to the title in the script that builds the database: https://github.com/RobLoach/libretro-database-dos

https://github.com/RobLoach/libretro-database-dos/blob/master/index.js#L79

**Comment by WizzardSK** [comment]
we already have the snaps in the form of "game (year)". so we should also change that or just change the database.

File: Named_Boxarts/$100,000 Pyramid (1988).png
Comment (RobLoach) [inline Named_Boxarts/$100,000 Pyramid (1988).png:1]: `$` must be `_`.... Here's the list of characters that have to be `_`...

```
&*/:`<>?\|"
```

**Comment by WizzardSK** [comment]
Okay, fixed that

File: Named_Boxarts/$100,000 Pyramid (1988).png
Comment (WizzardSK) [reply Named_Boxarts/$100,000 Pyramid (1988).png:1]: replaced $ and & characters

**Pull request merged**